examples/hello-world/Makefile.c64.defines

```diff
@@ -0,0 +1,2 @@
+DEFINES = WITH_CLIENT,WITH_DNS,WITH_PFS,WITH_LOGGING
```

Comment:
At least I would keep

DEFINES = WITH_LOGGING

as one can not know that hello-world (or any example) requires networking as provided by additional hardware or by vice hardware  emulation.

At least there would be an output on screen with the reason for the problem (not memory critical for hello world I think).

On the other hand one can not assume that the user has networking support (either by hardware or by emulation). 

So either the emulator or the original C64 has to provide network support ... or one has to write its own C64 platform for contiki.

For hello-world I think that this Info is very useful (also with the increased memory usage for hello-world) :

![helloworlderrorinfo](https://cloud.githubusercontent.com/assets/8344035/6255067/954f2d70-b7ad-11e4-828f-454f40a96aea.png)

Just my opinion, but also can be removed for this PR if requested.